tags/create.tag

```diff
@@ -64,23 +73,50 @@
             projectId: curProjectId.id
           };
           curProjectId.set(projectData);
-
+          // update current project to Users collection id document
+          // let refCurProjectId = this.db.doc('Users/' + userId);
+          // refCurProjectId.update({curProjectId: curProjectId.id});
+          this.db.doc('Users/' + userId).update({curProjectId: curProjectId.id});
           // trigger to pass curProjectId
-
           observer.trigger('project:created', curProjectId.id, this.inputProjectTitle);
-          //trigger to pass project name
-          observer.trigger('project:name', this.inputProjectTitle);
-
           showDialog = false;
           this.refs.projectTitle.value = '';
           this.showProjectTitle = true;
+          this.newProject = true;
+          this.firstProject = false
         }
         else{
           throw new Error('User is not signed in - should not see create tag');
         }
       }
     }
+    //
+    observer.on('project:firstProject', (firstProject) => {
+      this.firstProject = true;
+      this.newProject = false;
+      this.update();
+    });
+
+    observer.on('project:newProject', (newProject) => {
+      this.newProject = true;
+      this.firstProject = false
+      this.update();
+    });
 
+    // check if have Projects
+    checkProject(){
+      this.projectCollection = database.collection('Users/').doc(firebase.auth().currentUser.uid).collection('Projects');
+      this.projectCollection.get().then((querySnapshot) => {
+        if (querySnapshot.empty === true) {
+          this.firstProject = true;
+          observer.trigger('project:firstProject', this.firstProject);
+        }
+        else {
+          this.newProject = true;
+          observer.trigger('project:newProject', this.newProject);
+        }
+      });
+    }
```

Comment:
if we have this function (`checkProject`), why do we need the observers to set firstProject?

Reply:
Yes, I thought I need it in two places but it's not necessary.